storybookjs/storybook
# Replace old intro animation with new animation

Made by @iDanbo! 🙏

## What I did
Replace the old Storybook intro animation with a newer one that mirrors the visual design and animation style we have now.

## How to test

- View the [README](https://github.com/storybooks/storybook/tree/update-intro-gif), and check that the new video (with the updated logo) appears and starts playing
